collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java

```diff
@@ -362,35 +359,30 @@ public void onClick(View v) {
             //
             // Text upper left; audio upper right; video below audio on right.
             // image below text, audio and video buttons; left-aligned with text.
-            textParams.addRule(RelativeLayout.ALIGN_PARENT_LEFT);
+            if (isNeedPatchRTL()) {
```

Comment:
I don't understand this fragment `ALIGN_PARENT_START` and `ALIGN_PARENT_LEFT` should be the same so they are used under different circumstances?

Reply:
No, I just had a temporary black hole in my head and didn't catch it.